### ISSUE TITLE
Docker pull before each new task

### ISSUE THREAD

**Issue by kelson42** [openzim/zimfarm#201]
Might be good to just make a pull on the docker image before starting a new task. We have remarked that the mwoffliner 1.8.6 was in fact using the mwoffliner 1.8.5... so I have rebuilt the docker image just to fix that... but now it looks like we have no way to deploy it automatically

**Comment by automactic** [comment]
Already does this. See [run_mwoffliner.py#L27](https://github.com/openzim/zimfarm/blob/master/worker/app/operations/run_mwoffliner.py#L27)

To be specific: the version of the offliner comes from the version stored in the schedule object when the schedule is queued. (This logic can definitely be improved in the future)
